assets/services.json

```diff
@@ -1388,11 +1388,18 @@
         "||line-apps.com^",
         "||line-cdn.net^",
         "||line-scdn.net^",
+        "||line.biz^",
         "||line.me^",
         "||line.naver.jp^",
         "||linecorp.com^",
+        "||linefriends.com^",
+        "||linefriends.com.tw^",
+        "||linegame.jp^",
+        "||linemobile.com^",
         "||linemyshop.com^",
-        "||lineshoppingseller.com^"
+        "||lineshoppingseller.com^",
+        "||linetv.tw^",
+        "||nhncorp.jp^"
```

Comment:
Hello @rootmelo92118
I think `||nhncorp.jp^` must not be here, LINE is just one of their products.

Reply:
Solved